tools/Dockerfile

```diff
@@ -65,24 +65,16 @@ USER root
 RUN mkdir -p /usr/share/licenses/eksctl-anywhere && \
     chown -R builder:builder /usr/share/licenses/eksctl-anywhere
 
-ARG EKSA_VERSION=0.16.0
-ARG EKSA_SOURCE_URL="https://github.com/aws/eks-anywhere/archive/refs/tags/v${EKSA_VERSION}.tar.gz"
+# Build the latest EKS-A from main (we may want to change this to get a specific hash)
+ARG EKSA_SOURCE_URL="https://github.com/aws/eks-anywhere/archive/refs/heads/main.tar.gz"
```

Comment:
I think we should keep some way to change the revision we want to build from. Default to some stable released version.

Reply:
> Default to some stable released version.

I may have misunderstood, but I thought we could not set this to the released version. At least for now while we are trying to pick up k8s 1.28 support.

I can update this to use a variable, but please clarify what you think it should be pulling.

Reply:
For regular releases of testsys images, we should still be building EKS-A from stable releases versions. We want to allow the possibility of overriding the variable so we can build from any arbitrary git revision for special testing, e.g. new K8s version testing.

Reply:
This would also ideally mean, we need some way of specifying that through `make` and plumbing it to `docker build`